lithops/storage/backends/aws_s3/aws_s3.py

```diff
@@ -51,27 +64,30 @@ def __init__(self, s3_config):
                 retries={'max_attempts': OBJ_REQ_RETRIES}
             )
             self.s3_client = boto3.client(
-                's3', aws_access_key_id=self.access_key_id,
-                aws_secret_access_key=self.secret_access_key,
-                aws_session_token=self.session_token,
+                's3', aws_access_key_id=s3_config["aws"]["access_key_id"],
+                aws_secret_access_key=s3_config["aws"]["secret_access_key"],
+                aws_session_token=s3_config["aws"].get("session_token"),
                 config=client_config,
                 region_name=self.region_name
             )
         else:
+            logger.debug("Creating default boto3 client")
             client_config = Config(
-                signature_version=UNSIGNED,
```

Comment:
For now I will keep the default client as is, so that it can get credentials from the AWS role in the lambda runtime. If any user needs unsigned requests, it should explicitly create an s3 client or specify it in the s3 config

Reply:
We have an app that is analyzing a AWS S3 public bucket, where the client is automatically created by lithops because we put in `iterdata` a bucket address, [like in this example](https://github.com/lithops-cloud/lithops/blob/master/examples/map_cos_prefix.py) but for AWS S3, so we should find a way to keep this unsigned flag in the case that a user does not have credentials at all in his machine. If the user has credentials in his machine, the unsigned flag is not necessary.